python/running.md

```diff
@@ -11,18 +13,17 @@ $ source /cvmfs/sft.cern.ch/lcg/views/setupViews.sh LCG_94python3 x86_64-slc6-gc
 
 If you have a computer running MacOS or some Linux distribution, it will have
 come will Python pre-installed. Either way, a simple way to get Python on your 
-computer is to install [Anaconda][anaconda].
+computer is to install [Anaconda][anaconda]. In any case, it is highly recommended to use a virtual environment, be it with conda or other similar packages, and not your system installation.
 
 {% callout "Python 2 or 3?" %}
 
 You might see material that talks about Python 3 or Python 2. Like a lot of other software, 
 Python is regularly updated and groups batches of updates, including bug fixes 
-and new features, into versions. The interesting thing about Python 3 is that 
-it isn’t _backwards compatible_ with Python 2. This means that code that works 
+and new features, into versions. Python 3 is a new major release and is not _backwards compatible_ with Python 2, which is an exceptional break and not intended to happen again with Python 4. This means that code that works 
 when run with version 2 of Python may not necessarily work when run with 
-version 3. Python 2 was around for a long time, and so the process of migrating to Python 3 has been slow, which is why so many people talk about it. However the support for Python 2 will soon end, so now everyone is encouraged to use Python 3. That is why in this lesson, we will use Python 3.
+version 3. Python 2 was around for a long time and the process of migrating to Python 3 has been slow. However the support for Python 2 will soon end (2 month, in 2020!) and with it, most packages completely drop the support or already have.
 
-Note that, for instance, Alice and LHCb software is not (yet) compatible with Python 3. We strongly encourage you to always use Python 3 and only switch to Python 2 when you have to (e.g. when you need to use some software which is not compatible with Python 3). You can install both Python 2 and Python 3 using 
+Note that, for instance, Alice and LHCb software is not (yet) fully compatible with Python 3. We strongly encourage you to always use Python 3 and only switch to Python 2 when you have to (e.g. when you need to use some software which is not compatible with Python 3). You can install both Python 2 and Python 3 using 
 [Anaconda](https://www.anaconda.com/distribution/).
```

Comment:
I think we should suggest miniforge to avoid Anaconda incompatibilities:
```suggestion
[conda](https://github.com/conda-forge/miniforge#download).
```
Alternatively, a less extreme option would be to suggest [miniconda](https://docs.conda.io/en/latest/miniconda.html).